server/cmwell-spa/utils/upload.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+tokenFileName=~/dev/uploadTool/token-$1
```

Comment:
Should get from arg. Maybe default to local file named e.g. `token`, and if global default is needed, I think `~/.cmwell/` could be a directory to store such meta data for all our tools.

Reply:
:+1: